src/main/java/com/github/princesslana/eriscasper/rx/websocket/RxWebSocket.java

```diff
@@ -51,11 +50,13 @@ private Listener(ObservableEmitter<RxWebSocketEvent> em) {
     @Override
     public void onClosed(WebSocket ws, int code, String reason) {
       em.onNext(ClosedTuple.of(ws, code, reason));
+      em.onComplete();
```

Comment:
Are we sure there's no `complete()` method? onComplete seems more like a runnable function rather than a method which actually closes the observable.

Reply:
Gotcha.